README.md

```diff
@@ -11,6 +11,10 @@ Requirements
 - [SFML 2.5](http://www.sfml-dev.org/download.php)
 - [CSFML 2.5](http://www.sfml-dev.org/download/csfml/)
 
+Usage
```

Comment:
'Usage' makes it sound like the wiki is a comprehensive guide on how to use rust-sfml, but it's mostly a guide for setting up rust-sfml on different platforms if you're having difficulties doing so.